cnxarchive/scripts/export_epub/modeling.py

```diff
@@ -25,8 +25,11 @@
     )
 
 
-def document_factory(ident_hash, context=None):
+def document_factory(ident_hash, context=None, baked=False):
```

Comment:
I think you need `baked=None` or your check below should be `if baked and context is False`, otherwise it will always be False and the RuntimeError will never be raised.